### ISSUE TITLE
suggestion for DV1/appName.txt

### ISSUE THREAD

**Issue by alipartovi1990** [Alireza89/COEN-6311-Project#2]
we should choose a name for our application

**Comment by Alireza89** [comment]
I suggest "vegan record"

**Comment by Alireza89** [comment]
or "Vegan's impact"

**Comment by alipartovi1990** [comment]
not bad , how about green house reducer

**Comment by Alireza89** [comment]
> not bad , how about green house reducer

It is too wordy

**Comment by alipartovi1990** [comment]
> or "Vegan's impact"

i think is not bad

**Comment by Alireza89** [comment]
> > or "Vegan's impact"
> 
> i think is not bad

but i don't like the apostrophe s in the name

**Comment by alipartovi1990** [comment]
consume vegtable reduce green house
CVRG

**Comment by alipartovi1990** [comment]
> I suggest "vegan record"

i think is a good  name

**Comment by alipartovi1990** [comment]
vegan calculator ? what do you think?

**Comment by Alireza89** [comment]
> vegan calculator ? what do you think?

it is a good name but is not original. I found this on the web: https://thevegancalculator.com/#calculator

**Comment by Alireza89** [comment]
What about  VIE: "Vegan Impact on the Environment"
in french, la vie means the life!

**Comment by alipartovi1990** [comment]
perfect . it is a final name
